ekncontent/docs/reference/ekncontent/Makefile.am

```diff
@@ -1,73 +1,43 @@
-## Process this file with automake to produce Makefile.in
+DOC_MODULE = eos-knowledge-content
+DOC_MODULE_VERSION = @EKN_CONTENT_API_VERSION@
 
-# The name of the module, e.g. 'glib'.
-DOC_MODULE=eos-knowledge-content
+DOC_MAIN_SGML_FILE = $(DOC_MODULE)-docs.xml
 
-# Uncomment for versioned docs and specify the version of the module, e.g. '2'.
-DOC_MODULE_VERSION=@EKN_CONTENT_API_VERSION@
+DOC_SOURCE_DIR = \
+	$(top_srcdir)/ekncontent \
+	$(top_builddir)/ekncontent
 
-# The top-level SGML file. You can change this if you want to.
-DOC_MAIN_SGML_FILE=$(DOC_MODULE)-docs.xml
+SCANGOBJ_OPTIONS =
+SCAN_OPTIONS = --rebuild-types
+MKDB_OPTIONS = --xml-mode --output-format=xml
+MKTMPL_OPTIONS =
+MKHTML_OPTIONS =
+FIXXREF_OPTIONS =
 
-# Directories containing the source code.
-# gtk-doc will search all .c and .h files beneath these paths
-# for inline comments documenting functions and macros.
-# e.g. DOC_SOURCE_DIR=$(top_srcdir)/gtk $(top_srcdir)/gdk
-DOC_SOURCE_DIR=$(top_srcdir)/ekncontent $(top_builddir)/ekncontent
-
-# Extra options to pass to gtkdoc-scangobj. Not normally needed.
-SCANGOBJ_OPTIONS=
-
-# Extra options to supply to gtkdoc-scan.
-# e.g. SCAN_OPTIONS=--deprecated-guards="GTK_DISABLE_DEPRECATED"
-SCAN_OPTIONS=--rebuild-types
-
-# Extra options to supply to gtkdoc-mkdb.
-# e.g. MKDB_OPTIONS=--xml-mode --output-format=xml
-MKDB_OPTIONS=--xml-mode --output-format=xml
-
-# Extra options to supply to gtkdoc-mktmpl
-# e.g. MKTMPL_OPTIONS=--only-section-tmpl
-MKTMPL_OPTIONS=
-
-# Extra options to supply to gtkdoc-mkhtml
-MKHTML_OPTIONS=
-
-# Extra options to supply to gtkdoc-fixxref. Not normally needed.
-# e.g. FIXXREF_OPTIONS=--extra-dir=../gdk-pixbuf/html --extra-dir=../gdk/html
-FIXXREF_OPTIONS=
-
-# Used for dependencies. The docs will be rebuilt if any of these change.
 BUILT_HFILES=
-HFILE_GLOB=$(top_srcdir)/ekncontent/*.h
-CFILE_GLOB=$(top_srcdir)/ekncontent/*.c
-
-# Extra header to include when scanning, which are not under DOC_SOURCE_DIR
-# e.g. EXTRA_HFILES=$(top_srcdir)/contrib/extra.h
-EXTRA_HFILES=
+HFILE_GLOB = $(top_srcdir)/ekncontent/*.h $(top_builddir)/ekncontent/*.h
+CFILE_GLOB = $(top_srcdir)/ekncontent/*.c
 
-# Header files or dirs to ignore when scanning. Use base file/dir names
-# e.g. IGNORE_HFILES=gtkdebug.h gtkintl.h private_code
-IGNORE_HFILES=eknc-enums.h
+EXTRA_HFILES =
 
-# Images to copy into HTML directory.
-# e.g. HTML_IMAGES=$(top_srcdir)/gtk/stock-icons/stock_about_24.png
-HTML_IMAGES=
+IGNORE_HFILES = \
+	eknc-enums.h \
+	eknc-content-object-model-private.h \
+	eknc-database-manager-private.h \
+	eknc-domain-private.h \
+	eknc-media-object-model-private.h \
+	eknc-utils-private.h
```

Comment:
Adding this introduced one more documentation failure - we need to remove `eknc_utils_id_get_hash` from the sections.txt file, since it's in a private header file.

Reply:
Fixed